js/models.js

```diff
@@ -216,21 +211,21 @@ export function applyLocks() {
     if (wasLocked === isLocked) return;
     model.set('_isLocked', isLocked);
   });
+  log();
```

Comment:
empty log?

Reply:
It calls the function beneath is. I'll change the name.